site/app/libraries/database/DatabaseQueries.php

```diff
@@ -4066,7 +4066,7 @@ public function insertNewRegradeRequest(GradedGradeable $graded_gradeable, User
         try {
             $this->course_db->query("INSERT INTO regrade_requests(g_id, timestamp, $submitter_col, status, gc_id) VALUES (?, current_timestamp, ?, ?, ?)", $params);
             $regrade_id = $this->course_db->getLastInsertId();
-            $this->insertNewRegradePost($regrade_id, $sender->getId(), $initial_message);
+            $regrade_post_id = $this->insertNewRegradePost($regrade_id, $sender->getId(), $initial_message, $gc_id);
```

Comment:
can you remove the var `regrade_post_id` if its not used in the function?

Reply:
`insertNewRegradePost` returns the _post_id_ that's why i store it in a variable even if it's not used

Reply:
I dont think you need to store the post_id just because its returned by the called function... let's remove this one... thanks:)